test/e2e/multicast_test.go

```diff
@@ -556,7 +579,12 @@ func testMulticastForwardToMultipleInterfaces(t *testing.T, data *TestData, send
 	}
 }
 
-func runTestMulticastBetweenPods(t *testing.T, data *TestData, mc multicastTestcase, nodeMulticastInterfaces map[int][]string) {
+func runTestMulticastBetweenPods(t *testing.T, data *TestData, mc multicastTestcase, nodeMulticastInterfaces map[int][]string, checkReceiverRoute bool) {
+	currentEncapMode, _ := data.GetEncapMode()
+	if requiresExternalHostSupport(mc) && currentEncapMode == config.TrafficEncapModeEncap {
+		t.Logf("Multicast does not support using hostNetwork Pod to simulate the external host with encap mode, skip the case")
```

Comment:
Use `t.Skip`, then the subtest will be marked as "Skip" instead of "Pass"

Reply:
updated.